fsthttp/request.go

```diff
@@ -116,12 +124,12 @@ func newClientRequest() (*Request, error) {
 		return nil, fmt.Errorf("get client request and body: %w", err)
 	}
 
-	method, err := abiReq.GetMethod()
+	method, err := abiReq.GetMethod(RequestLimits.maxMethodLen)
```

Comment:
I wonder if we should put request and/or response `Limits` fields inside the `Request` struct, and pull from that instead.

We could still keep the global `RequestLimits` and `ResponseLimits` values, and a newly-constructed `Request` would have its fields simply be a copy of the global ones, but it'd also give you the ability to make adjustments per-request.

The main upside to this is that if you know one particular backend request needs an especially big buffer you can avoid allocating it unnecessarily for all the other requests too.

Reply:
I'm happy to explore other venues but currently we prefetch everything before handling the incoming request to the caller, and it is at this point that we need to tweak the various limits.   For outgoing requests (created by `NewRequest`), they don't apply AFAICT.

Reply:
Related to this, maybe it'd be better to name these `GlobalRequestLimits` and `GlobalResponseLimits` should we add per-requests ones in the future? Thoughts?